lib/internal/http2/core.js

```diff
@@ -2877,7 +2877,11 @@ function initializeOptions(options) {
 
 function initializeTLSOptions(options, servername) {
   options = initializeOptions(options);
-  options.ALPNProtocols = ['h2'];
+  if (options.ALPNProtocols) {
+    options.ALPNProtocols.push('h2');
+  } else {
+    options.ALPNProtocols = ['h2'];
+  }
```

Comment:
It’s not really obvious to me how this relates to the rest of the changes here, and it seems like something that could use tests of its own?

Reply:
I've removed that. It was a mistake. It was from another change I was working on in another branch that accidentally ended up in this branch.